src/decays/specializations/Ah/decay_Ah_to_AZ.inc

```diff
@@ -23,6 +23,7 @@ double CLASSNAME::get_partial_width<PseudoscalarHiggs, Photon, ZBoson>(
       res *= alpha_em_0/alpha_em;
    }
 
+   std::string tag = field_as_string<Higgs>(in_idx) + "-" + field_as_string<Photon>(out1_idx) + "-" + field_as_string<ZBoson>(out2_idx);
    if (flexibledecay_settings.get(FlexibleDecay_settings::print_effc_block)) {
       effhiggscouplings_block_input.push_back(
          {
```

Comment:
Could you please add a comment here, where these magic numbers (36,10,22,23) come from? One could alternatively move them into local variables with appropriate names.

Reply:
I've replace this weird formula on 0b7454b155455c06b70a0088324e778152f08521. Maybe now it's more readable?

Reply:
This is much better! Now even I understand what this was about. :)